src/credssp/mod.rs

```diff
@@ -676,6 +676,7 @@ impl SspiImpl for SspiContext {
             SspiContext::Ntlm(ntlm) => {
                 let auth_identity = match builder.auth_data {
                     Some(Credentials::AuthIdentity(identity)) => Some(identity),
+                    #[allow(unreachable_patterns)]
```

Comment:
question: If it’s unreachable, what about removing the whole branch?

suggestion: If you really want to keep this suppression, use `expect` so we get a warning when it does not apply anymore.